bluepysnap/circuit_validation.py

```diff
@@ -418,33 +472,43 @@ def _check_edges_population(edges_dict, nodes):
     Returns:
         list: List of errors, empty if no errors
     """
-    POPULATION_DATASET_NAMES = [
-        'edge_type_id', 'source_node_id', 'target_node_id', 'edge_group_id', 'edge_group_index']
     errors = []
     edges_file = edges_dict.get('edges_file')
     with h5py.File(edges_file, 'r') as h5f:
         edges = _get_h5_data(h5f, 'edges')
         if not edges or len(edges) == 0:
             errors.append(fatal('No "edges" in {}.'.format(edges_file)))
             return errors
+
         for population_name in edges:
+            required_datasets = ['edge_type_id', 'source_node_id', 'target_node_id']
+
             population_path = '/edges/' + population_name
             population = h5f[population_path]
+            groups = _get_edge_population_groups(population)
+
+            if len(groups) != 1:
```

Comment:
In case there are no groups and `len`==0? Don't we already have this check inside of `_check_edge_group_id`?

Reply:
Actually not. I need to take the "no group" case into account. 
This can maybe happens and should be a valid use case (you just want want the graph topology).